xml/deploy_cephadm.xml

```diff
@@ -757,6 +757,151 @@ o- time_server ................................................ [enabled]
+         <listitem>
+           <para>
+             A strong initial authentication when the connection is established
+             and full encryption of all post-authentication traffic, including
+             a cryptographic integrity check.
+           </para>
+         </listitem>
+       </varlistentry>
+     </variablelist>
+     <para>
+       For most connections, there are options that control which modes are used:
+     </para>
+     <variablelist>
+       <varlistentry>
+         <term>ms_cluster_mode</term>
+         <listitem>
+           <para>
+             Is the connection mode (or permitted modes) used for intra-cluster
+             communication between &ceph; daemons. If multiple modes are listed,
+             the modes listed first are preferred.
+           </para>
+         </listitem>
+       </varlistentry>
+       <varlistentry>
+         <term>ms_service_mode</term>
+         <listitem>
+           <para>
+             Is a list of permitted modes for clients to use when connecting to
+             the cluster.
+           </para>
+         </listitem>
+       </varlistentry>
+       <varlistentry>
+         <term>ms_client_mode</term>
+         <listitem>
+           <para>
+             Is a list of connection modes, in order of preference, for clients
+             to use (or allow) when talking to a &ceph; cluster.
+           </para>
+         </listitem>
+       </varlistentry>
+     </variablelist>
+     <para>
+       There are a parallel set of options that apply specifically to monitors,
+       allowing administrators to set different (usually more secure) requirements
+       on communication with the monitors.
+     </para>
+     <variablelist>
+       <varlistentry>
+         <term>ms_mon_cluster_mode</term>
+         <listitem>
+           <para>
+             Is the connection mode (or permitted modes) to use between monitors.
+           </para>
+         </listitem>
+       </varlistentry>
+       <varlistentry>
+         <term>ms_mon_service_mode</term>
+         <listitem>
+           <para>
+             Is a list of permitted modes for clients or other &ceph; daemons
+             to use when connecting to monitors.
+           </para>
+         </listitem>
+       </varlistentry>
+       <varlistentry>
+         <term>ms_mon_client_mode</term>
+         <listitem>
+           <para>
+             Is a list of connection modes, in order of preference, for clients
+             or non-monitor daemons to use when connecting to monitors.
+           </para>
+         </listitem>
+       </varlistentry>
+     </variablelist>
+     <para>
+       In order to enable msgr2 encryption mode during the deployment, you need
+       to add some configuration options to the &cephsalt; configuration before
+       running <command>ceph-salt apply</command>.
+     </para>
+     <para>
+       To enable the v2 service:
+     </para>
+<screen>&prompt.smaster;ceph mon enable-msgr2</screen>
```

Comment:
Not sure this will work. We haven't run `ceph-salt apply` so we haven't deployed or bootstrapped the cluster yet. So there wont be a cluster to run this on.

Reply:
@matthewoliver this was part of the upstream docs. Can remove? But how do we enable?

Reply:
Removed for the time being.